docs/SQLModelingStandard.md

```diff
@@ -0,0 +1,125 @@
+    `consent` is not sufficiently  informative to rule out user consents and 
+    examiner consents. A table containing the consent types would be called 
+    as such `consent_type`.*
+- A table that is **specific to a module** should be **prefixed** with the 
+  module name.
+  - *i.e. A table named `xyz` used in module `abc` should be name `abc_xyz`.*
+- **Acronyms** should be avoided, however may be used when the acronym would 
+  be **commonly used by the neuroimaging community**. The acronym should
+  **only contain upper case letter**. For historic reasons, “PSC” to refer to 
+  a Site is an exception to this rule.
+- When creating a table representing a **relation between two or more tables**. 
+  Regardless of the relation type (many-to-many, one-to-many, many-to-one or 
+  one-to-one), the table should have a composite name as such 
+  `table1_table2_rel`.
+  - *i.e. the table mapping users to their addresses would be named 
+  `user_address_rel` where `user` is the name of the Users' entity table and 
+  `address` is the name of the Addresses' entity table.*
+
+### Fields
+
+- The field name should be in **CapitalizedWords**.
+  - The field name should **start** with an **Upper case letter**.
+  - The field name should **not contain any underscore**.
+- The field name should be in **singular form**.
+- The field name should contain **English words** only.
+- The field name should **not contain abbreviations**.
+- **Acronyms** should be avoided, however may be used when the acronym would 
+  be **commonly used by the neuroimaging community**. The acronym should 
+  **only contain upper case letter**.
+- The **primary key** field of a table should be named `<TableName>ID`. While 
+  the table name is snake_case, the field names referencing it (such as the 
+  primary key) should follow field naming conventions. (ie. `my_table`’s 
+  primary key should be `MyTableID`).
+  - *i.e. the name of the primary identifier of the `country` table should be 
+  `CountryID`.*
+- When an explicit auto-incremented primary key field is present, it should be
+  of type unsigned int. Most tables should have an explicit primary key field.
+- When adding a field which serves as a **foreign key to another table**
+  in the database, the field should follow the same convention as above
+  `ReferenceTableNameID`.
+  - *i.e. The `country` table would have a primary identifier field named
+  `CountryID` and another table a foreign key reference to the `country`
+  table with a field named `CountryID`.*
+- When two(2) fields refer to the **same foreign key id**, a qualifier should
+  be added to the names.
+  - *i.e. Two fields pointing to the `CountryID` in the same table should
+  be named like in `OriginCountryID` and `DestinationCountryID`.*
+- Both **primary key and foreign key** fields should end with **`ID`**
+  in upper case.  - **Date** fields should not just be named “Date”. A
```

Comment:
There's a linebreak missing here too

Reply:
Look like it was lost in the twilight zone, restored